travis-run-tests.sh

```diff
@@ -1,29 +0,0 @@
-#! /usr/bin/env bash
-
-current_version=`python setup.py --version`
-
-make test
-
-# This part is executed only when merging a branch in master.
-if [[ "$TRAVIS_BRANCH" == "master" && "$TRAVIS_PULL_REQUEST" != false ]]; then
-    # A version bump is requested by default, except:
-    # - when changing a Markdown (.md) file
-    # - or when only blank lines are added/removed
-
-    # Note: git diff-index does not work with --ignore-blank-lines option.
-    bumping_changes=`git --no-pager diff --ignore-blank-lines master openfisca_france | grep "^diff" | grep -v "\.md$"`
```

Comment:
Yes, it is.
At the same time, we also have a more understandable script, which complies to the `country-template` standard (and does't rely on porcelain) 🤔.